packages/mdc-tabs/tab-bar/_mixins.scss

```diff
@@ -2,7 +2,7 @@
 
 @mixin mdc-tab-bar-indicator-ink-color($color) {
   .mdc-tab-bar__indicator,
-  &:not(.mdc-tab-bar-upgraded) .mdc-tab::after {
+  &.mdc-tab-bar:not(.mdc-tab-bar-upgraded) .mdc-tab::after {
```

Comment:
HMmmmmmmmmm...this is tricky

Remind me, why do we use `.mdc-tab::after` and not `.mdc-tab-bar__indicator` for the CSS only version of tabs?

Reply:
Going to look into removing the .mdc-tab::after element in favor of .mdc-tab-bar__indicator. Will open pre-pr if possible